packages/lambda/package.json

```diff
@@ -29,7 +29,7 @@
     "prompt": "^1.0.0",
     "resolve-tree": "^0.1.13",
     "semver": "^6.0.0",
-    "serverless-http": "^1.6.0",
+    "serverless-http": "^2.0.0",
```

Comment:
There's already `2.0.1` available. 🤷‍♂️

Reply:
Yes, and since we specified a range, we're already using it: https://github.com/xing/hops/pull/856/files#diff-8ee2343978836a779dc9f8d6b794c3b2R11000